### DISCUSSION TITLE
Availability of Pre Made PC boards and Case?

### DISCUSSION THREAD

**Discussion by tkiller99999** [gusmanb/logicanalyzer#24] [General]
OK.. So call me Lazy....   But

Do you have premanufactured   PC boards?  and the Printed Case available?

I know I can send off the design files to PCBWay or another PC factory... but I would rather the creator gets the Money.

Ted Killmeyer

**Comment by gusmanb** [comment]
Hi.

Sorry but no, I don't have premade boards. If you want to order some I recommend you JLCPCB, they are very cheap and for now I had no issues with them.

Cheers.

**Comment by Eebigdog** [comment]
Is a BOM available? Esp for the connectors used and where to get them?

**Comment by mpictor** [comment]
I too was hoping for a way to purchase a PCB rather than having to deal with MOQ and offshore shipping times. Anyone know if there's a service that does this for small things? I'm guessing it's too small (and/or too unassembled) for massdrop and crowdsupply.

**Reply by gusmanb** [reply]
JLCPCB offers really cheap PCBs and even has an assembly service, I always use it. You can order from 5 to as many as you want, and 5 PCB's cost 2$. The service is very cheap but the shipping is around 25-30€ (unless you use cainiao but I would warn you that it takes ages to get things and is not the most reliable service...)

**Comment by tkiller99999** [comment]
I have several boards (3 of them)  extra….

 

So if you want I would sell you one…10 dollars + shipping.

 

 

 

 

 

 

Regards,

Ted Killmeyer

 

___________________________

Theodore Paul Killmeyer Jr.



(USS Nevada SSBN 733 Plank Owner)

Computer Engineer

Electrical Engineer

Senior Systems Architect

Skype:  Theodore.Killmeyer

Email:  ***@***.***> ***@***.***

Email:  ***@***.***> ***@***.***

Email:  ***@***.***> ***@***.***

Mobile: +1 (720) 937-0281

**Reply by cnsmdp** [reply]
Hi Ted,

Do you still have those extra boards, out of luck? Thanks!

Mathieu

**Reply by Jwildesbr** [reply]
Hi Ted,

I am interested in some PCBs too... is still available?

Best regards,

Jwildesbr

**Reply by tkiller99999** [reply]
sorry No the boards are no longer available...  I am trying to get the Version 2 boards...

**Comment by arekm** [comment]
pcbway I think has a site where projects can be published and people can order hardware easily (if I understand correctly thing behind https://www.pcbway.com/project/shareproject/). Probably jlpcb has something similar.

Maybe logicanalyzer could be provided as such project, so just click and go with ordering hardware?

**Comment by PhilByte** [comment]
This design looks great and I would really like a better analyser. the 100MHz version seems ideal for me. 
Are the Gerber files available for download as I don't use Kicad?
I usually use Seed for PCB manufacture but I'm sure PCBway does a good job too. 
Hopefully the design fits in 100mm square.
I have previouly built a 24 x 32k channel 30MHz Logic analyser based on an old CPLD design whose designer seems not to respond now. It worked (mostly) but the software is not so good for these days. 
Thank you

**Reply by gusmanb** [reply]
If you are not in a hurry I would recommend you to wait a bit, I'm testing a new design ;)
I will receive the first ones assembled in a couple of weeks and if everything works ok I will show it here.

Else you have to export the Gerber files from KiCad, I don't include the gerber files because each manufacturer can have different requirements for the files and is better to ensure you export according to their parameters.

Cheers.

**Reply by p3t3rm0v** [reply]
Hi interested on purchase a pair fully assembled. Tryed contact you by linkedin bcause dont't know if you're still operative here. Both same area.

**Comment by PhilByte** [comment]
Many thanks for your reply - much appreciated.
I'll try the Kicad route but I don't really get on with it very well - I'm
a bit old school.
Good luck with the new design - if you get round/solve the Pi problem it
should be brilliant.
Phil

**Comment by PhilByte** [comment]
Following on:-
I have installed and used Kicad 8.0 to extract Gerber files.
The format seems suitable for my manufacturer as displayed by Viewmate.
I may have to change the layer names but I have to do that anyway with my
current software.
The boards seem to be v4.0 - is that currently correct and used with v5
software/firmware?
I'll wait a bit to see what transpires in the next few weeks but as I said
100MHz would be easily sufficient for my purposes.
Thanks
Phil

**Reply by gusmanb** [reply]
Yes, that's correct, has been a while since the last change in the hardware, all that has changed is the firmware/software.

Cheers.

**Comment by gusmanb** [comment]
If anyone is intereseted there is a new design that I'm testing and I will have some spares, they are fully assembled except for the pico itself. Check main page for a picture of the new design.

**Reply by kd7eir** [reply]
I am definitely interested in one of the spares.

**Reply by DeeJay** [reply]
Another expression of interest in a V6.0/RP2350 compatible breakout/buffer board in a "ready to run" format. Either as part of a testing effort, or as part of a production run.

**Reply by mpictor** [reply]
Likewise, I am also interested in one of the mostly-assembled boards, either in testing or as part of a production run.

**Reply by ORARadwan** [reply]
I am also interested in the new version and would love to get a populated PCB! please let us know when they would be available! Thank you for this great project!

**Reply by gusmanb** [reply]
@ORARadwan

If you check the last message is already arranged and all the spare ones have been reserved, but after these are sent if all goes ok I will start a tindie shop and will deliver these regularly.

Contact me through [logicanalyzerorders@gmail.com](mailto:logicanalyzerorders@gmail.com) and I will notify you when there are more available.

Cheers.

**Reply by tkiller99999** [reply]
Gusmanb,   Any update?   is there a Tindie for the Version 2 Boards?  I can do the SMC components, so I would just need a board and location for the components

**Reply by gusmanb** [reply]
I'm out in a business trip, will be back next week and will order a new batch.

Cheers.

**Reply by kiwisincebirth** [reply]
I to would be interested in ordering a preassembled board.

**Reply by sonicthewedge** [reply]
I would be interested in a preassembled board when V6 is available. Thanks.

**Reply by gusmanb** [reply]
@kiwisincebirth @sonicthewedge send me an email to logicanalizerorders@gmail.com and I will add you to the list.

**Comment by fbgentle1** [comment]
I am definitely interested in purchasing a board.
Once again thanks for all of your work on the analyser.
Cheers Bruce

**Comment by gusmanb** [comment]
This weekend will see how I can organize it, I will update this post with more info 😄 

Cheers

**Reply by peter-strom** [reply]
Wow, I just found out about this project and I'm also very interested in a kit.

**Reply by zJudGenie** [reply]
Same here, I'm a student and having a 200MHz cheap logic analyzer like this would be really handy, thank you

**Reply by sigmate** [reply]
That's great news! Count me in for a ready-made board, please! Thanks a lot!

**Comment by gusmanb** [comment]
Sorry for the delay, I'm really busy right now with my job, will try to sort things this weekend.

Cheers.

**Comment by tkiller99999** [comment]
Well I am trying to get the Version 2 BOM and Board files so I can make several for myself…  So at this time, sorry I have none to sell but the one I made in the begining

 

 

 

 

Regards,

Ted Killmeyer

 

___________________________

Theodore Paul Killmeyer Jr.



(USS Nevada SSBN 733 Plank Owner)

Computer Engineer

Electrical Engineer

Senior Systems Architect

**Comment by gusmanb** [comment]
Ok, I got things sorted.

I have 6 spare units, they come with all assembled except for the pico, you can add a pico or a pico 2, the design works with both.
It has the shifters integrated and a switch to select between 5v/3v3/ext voltage reference.
The design has been tested with signals up to 200Mhz with good results (better with Pico 1, the latching on the pico2 seems to affect a bit the response at such speeds, or maybe the new pads dessign is not as fast as the old pico ones).

Each unit will be 25€ plus shipping, payment through paypal, I will send the units on next friday after receiving payment.

To do the order contact me through logicanalyzerorders@gmail.com and we will arrange it.

I will send you the latest firmware and a link to the latest software version for your OS, so remember to state in the order:

- Version of the pico you plan to use (Pico, Pico W, Pico 2).
- State if you want the Turbo version of the firmware (more overclock and overvoltage but able to sample up to 400Msps in blast mode).
- OS where you plan to use it (only OS where I haven't tested the latest software is on MacOS X for ARM devices, keep it in mind).
- Address.

Once I receive the order will check the delivery costs to your address and will send you the total ammount to be paid (if you have a cheaper delivery option than the one I find I gladly will give you my address so you can arrange the package pickup).

And if this goes well I might open a store in tindie as I have some other projects that I will publish soon and can be good to have them available 😄 

Cheers!

@kd7eir  @DeeJay @fbgentle1 @peter-strom @zJudGenie @sigmate

**Comment by tkiller99999** [comment]
Ok I will contact you…   I might also want to just get the Version 2 PC board….   We will see when you get your Tindie shop up…

 

 

 

 

Regards,

Ted Killmeyer

 

___________________________

Theodore Paul Killmeyer Jr.



(USS Nevada SSBN 733 Plank Owner)

Computer Engineer

Electrical Engineer

Senior Systems Architect

Skype:  Theodore.Killmeyer

Email:  ***@***.***> ***@***.***

Email:  ***@***.***> ***@***.***

Email:  ***@***.***> ***@***.***

Mobile: +1 (720) 937-0281

**Comment by firew4lker** [comment]
I would also like to have one o the pre assembled new boards.

**Reply by gusmanb** [reply]
All have been sent but next week I will order more, contact me at [logicanalyzerorders@gmail.com](mailto:logicanalyzerorders@gmail.com) and I will send you an email when they are available.

Cheers.

**Comment by tkiller99999** [comment]
Cheers and thanks.

 

 

 

 

 

Regards,

Ted Killmeyer

 

___________________________

Theodore Paul Killmeyer Jr.



(USS Nevada SSBN 733 Plank Owner)

Computer Engineer

Electrical Engineer

Senior Systems Architect

Skype:  Theodore.Killmeyer

Email:  ***@***.***> ***@***.***

Email:  ***@***.***> ***@***.***

Email:  ***@***.***> ***@***.***

Mobile: +1 (720) 937-0281

**Comment by jeetwin-app** [comment]
Hi Ted, it’s great that you want to support the creator directly! Many project creators do offer pre-manufactured PCBs and cases for convenience. It’s worth checking the official documentation or project page for any store links or recommended sources.

**Comment by jeetwin-app** [comment]
Prefabricated PCBs and cases save time and effort, so it’s completely understandable to prefer that route. If the creator doesn’t sell them directly, sometimes they partner with suppliers or provide verified third-party options.

**Comment by jeetwin-app** [comment]
If you haven’t found pre-made options, you could consider reaching out to the creator directly. They might offer boards/cases informally or appreciate the feedback for future offerings. Good luck, and I hope you find the perfect solution!

**Comment by salmantino** [comment]
Me gustaría tener una placa premontada, vivo en España ¿podrias decirme precio incluido envio ?
